dask/blockwise.py

```diff
@@ -62,6 +62,16 @@ def get(self, idx: tuple[int, ...], default) -> Any:
         except KeyError:
             return default
 
+    @property
+    def delayed(self):
+        """Whether all dependencies are Delayed objects"""
+        return False
+
+    @property
+    def delayed_deps(self) -> set:
+        """Return Delayed dependencies"""
+        return set()
```

Comment:
Should probably remove the unnecessary `delayed` property and document the `delayed_deps` property better (and possibly call it `delayed_dependencies`?)